Sming/SmingCore/Network/TcpConnection.cpp

```diff
@@ -36,6 +36,12 @@ TcpConnection::~TcpConnection()
 		delete[] sslFingerprint.pkSha256;
 	}
 	freeSslClientKeyCert();
+	if(ssl_ext != NULL) {
```

Comment:
That is not the right place to put that change. It is also not needed because SSL closing will free internally also ssl_ext.